game_clean/copy/start_tf2.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+cd "$(dirname "$0")"
+
+ARGS="-steam -game tf -insecure -novid -nojoy -nosteamcontroller -nohltv -particles 1 -noborder -particle_fallback 2 -dev -nobreakpad -console"
+
+export LAUNCH_PREFIX="$HOME/.steam/bin32/steam-runtime/run.sh"
+LAUNCH_PREFIX+=" $(command -v mangohud || true)"
```

Comment:
Remove mangohud from here. Instead, allow users to customize their `LAUNCH_PREFIX` (script should add on `run.sh`).